tasks/bump.js

```diff
@@ -56,6 +56,16 @@ module.exports = function(grunt) {
         exactVersionToSet = false;
     }
 
+
+    //Use: --commitmsg="add me to commit message"
+    var appendToCommitMessage = grunt.option('commitmsg');
```

Comment:
maybe the commitMsg should accept a template-able string with the version number, if it already doesnt